frontend/homepage.html

```diff
@@ -32,6 +32,7 @@
                     <p>C++</p>
                 </div>
                 <div class="quiz-option">
+                    <a href="description.html">
```

Comment:
why did you add href link ?

Reply:
when the user click python quiz at home page it should redirect to the description page right? That's I gave href in home page.